src/measure_theory/borel_space.lean

```diff
@@ -674,6 +674,48 @@ begin
   exact is_measurable.Union (λ i, hf i (is_open_lt' _).is_measurable)
 end
 
+private lemma ae_measurable.is_lub_of_nonempty {ι} (hδ : nonempty δ) (hι : nonempty ι)
+  {μ : measure δ} [encodable ι] {f : ι → δ → α} {g : δ → α}
+  (hf : ∀ i, ae_measurable (f i) μ) (hg : ∀ b, is_lub {a | ∃ i, f i b = a} (g b)) :
```

Comment:
Is it possible to replace the forall assumption in `hg` with `∀ᵐ b ∂μ`? It would seem more natural in this setting. Same thing for all the other statement.